README.md

```diff
@@ -130,6 +130,7 @@ Third-party components (of the entity-component-system pattern).
 - [Particle System](https://github.com/IdeaSpaceVR/aframe-particle-system-component) - Particle systems (e.g., rain, dust, snow, fire)
 - [WebVR Controller](https://github.com/richardanaya/aframe-webvr-controller) - HTC Vive controllers
 - [Keyboard](https://github.com/richardanaya/aframe-keyboard) - Full VR keyboard for use with WebVR controllers
+- [Bmfont](https://github.com/bryik/aframe-bmfont-text-component) - Renders bitmap/SDF fonts for high quality 2D text
```

Comment:
yours seems more complete, but have you seen https://github.com/mattdesl/aframe-bmfont-text/blob/master/index.js? just making sure you saw